drivers/can/Kconfig.net

```diff
@@ -0,0 +1,38 @@
+# Kconfig.net - Configuration options for IPv6 over CAN
+
+#
+# Copyright (c) 2019 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config CAN_NET
+	bool "Enable 6loCAN network interface [Experimental]"
+	default n
```

Comment:
"default n" strings can be omitted because by default the boolean variables are false